Scenarios/BlueGreen-Deployment-for-AKS/README.md

```diff
@@ -1,12 +1,12 @@
 # Blue Green Deployment for AKS
 This architectural pattern describes how to properly implement a Blue-Green deployment of an AKS cluster that follows the guiding tenets of the [Azure Well-Architected Framework](https://docs.microsoft.com/en-us/azure/architecture/framework/). For Blue Green deployment at the application level, refer to [this article](https://docs.microsoft.com/en-us/azure/architecture/example-scenario/blue-green-spring/blue-green-spring).
 The main purpose is to provide a reliable and high availability solution when:
```

Comment:
Change to: "The main purpose of this pattern..."

Reply:
fixed